lib/numtheor.gi

```diff
@@ -1385,24 +1347,16 @@ end );
 
 
 InstallGlobalFunction(PValuation,function(n,p)
-local a,v;
-  if not IsPrimeInt(p) or not IsRat(n) then
+local v;
+  if not IsInt(p) or not IsRat(n) or p = 0 then
     Error("wrong parameters");
   fi;
```

Comment:
Yes. I hope somebody will make a PR improving it after this PR has been merged :-)